azure-iot-device/azure/iot/device/provisioning/pipeline/pipeline_stages_provisioning.py

```diff
@@ -69,15 +69,15 @@ def _form_complete_result(operation_id, decoded_response, status):
         return registration_result
 
     def _process_service_error_status_code(self, original_provisioning_op, request_response_op):
-        logger.error(
+        logger.info(
             "{stage_name}({op_name}): Received error with status code {status_code} for {prov_op_name} request operation".format(
                 stage_name=self.name,
                 op_name=request_response_op.name,
                 prov_op_name=request_response_op.request_type,
                 status_code=request_response_op.status_code,
             )
         )
-        logger.error(
+        logger.info(
```

Comment:
>info [](start = 15, length = 4)

I feel this (and the one above) could probably be debug - In general I feel like info should probably be higher level stuff than this, which means very little to the user.